CHANGELOG.md

```diff
@@ -2,6 +2,62 @@
 
 ## Unreleased
 
+### SDK
+
+* Added scope attributes to `InstrumentationScopeInfo` accessible
+  via `InstrumentationScopeInfo#getAttributes()`. Will add the ability to specify scope attributes
+  in Meter, Tracer, and Logger in a future version.
```

Comment:
Just double-checking things: are we fine with releasing this while this issue is still under discussion? https://github.com/open-telemetry/opentelemetry-specification/issues/2762